circle.yml

```diff
@@ -1,17 +1,28 @@
-machine:
-  environment:
-    MINICONDA_PATH: $HOME/miniconda
-    CONDA_ENV_NAME: testenv
+version: 2
 
-dependencies:
-  override:
-    - ./continuous_integration/circle/build_doc.sh:
-        timeout: 3600 # seconds
-test:
-  override:
-    - cat ~/log.txt && if grep -q "Traceback (most recent call last):" ~/log.txt; then false; else true; fi
-
-general:
-  artifacts:
-    - "doc/_build/html"
-    - "~/log.txt"
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.6.2
+    steps:
+      - checkout
+      - run:
+          command: ./continuous_integration/circle/build_doc.sh
+          no_output_timeout: 60m
+          environment:
+            MINICONDA_PATH: ~/miniconda
+            CONDA_ENV_NAME: testenv
+      - run: 
+          command: |
+            cat log.txt && \
+            if grep -q "Traceback (most recent call last):" log.txt; \
+            then false; \
+            else true; \
+            fi
+      - run:
+          command: |
+            mkdir -p /tmp/data
+            cp -R doc/_build/html /tmp/data/html
+            cp log.txt /tmp/data/log.txt
+      - store_artifacts:
+          path: /tmp/data
```

Comment:
Can you not use doc/_build/html as artifact rather than having to copy things around? Have a look at the scikit-learn config:
https://github.com/scikit-learn/scikit-learn/blob/e161700c6f8948b820073a72f09a71e1dda89ccc/.circleci/config.yml#L21-L26